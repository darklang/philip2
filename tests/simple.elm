-- this file left for illustrating purposes, the tests are actually using
-- the companion json file `simple.elm.json`
module Test exposing (..)

x : Int -> String
x v =
  toString v
