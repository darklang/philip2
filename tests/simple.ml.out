open Tea
open! Porting
let x (v : int) = (toString v : string)
